app/main.js

```diff
@@ -0,0 +1,6 @@
+(function(app) {
+    document.addEventListener('DOMContentLoaded', function() {
+        ng.core.enableProdMode();
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["ng" is not defined.](https://www.codacy.com/app/garethjackson/www-heptonstall-info/file/2734059927/issues/source?bid=3331202&fileBranchId=3351007#l3)